helpers/generate-index-html.js

```diff
@@ -202,12 +202,12 @@ li.selected {
 }
 @font-face {
     font-family: 'OpenMojiBlack';
-    src: url('font/OpenMoji-Black.ttf') format('truetype');
-}
+    src: url('font/glyf/OpenMoji-Black.ttf') format('truetype');
+  }
```

Comment:
Why adding two spaces to this line? I'd propose keeping it as-is:

```suggestion
}
```

Also, heads-up: whitespace in this file is changed in #428.

Reply:
You would simply need to rebase off of this in your PR for the changes to whitespace.

I agree that the whitespace is superfluous and will update the final commit to remove this. Thanks for reviewing!